ntbs-service/Pages/Notifications/Overview.cshtml

```diff
@@ -0,0 +1,28 @@
+@page "{handler?}"
+@model ntbs_service.Pages_Notifications.OverviewModel
+
+@{
+    Layout = "../Shared/_NotificationLayout.cshtml";
+    ViewData["Title"] = "Overview";
+    ViewData["NotificationId"] = @Model.NotificationId;
+}
+
+<partial name="./OverviewPartials/_PatientOverviewPartial" model="@Model"/>
+
+<partial name="./OverviewPartials/_EpisodeOverviewPartial" model="@Model"/>
+
+<partial name="./OverviewPartials/_ClinicalDetailsOverviewPartial" model="@Model">
+
+<partial name="./OverviewPartials/_ContactTracingOverviewPartial" model="@Model">
+
+<partial name="./OverviewPartials/_SocialRiskFactorsOverviewPartial" model="@Model">
+
+<partial name="./OverviewPartials/_TravelOverviewPartial" model="@Model">
+
+<partial name="./OverviewPartials/_ComorbiditiesAndImmunosuppresionOverviewPartial" model="@Model">
+
+<partial name="./OverviewPartials/_PreviousHistoryOverviewPartial" model="@Model">
+
+<back-to-top-button inline-template>

```

Comment:
As it's javascript-only, we don't want to be showing this button without javascript. We could use `v-if`, or not use an `inline-template` and instead provide the template in javascript (see `template` property on https://vuejs.org/v2/guide/components.html#Base-Example)
Alternatively, we could implement the scroll back up with a link to the top of the page ( `<a href='#top'>` where `top` is the id of an element), which is what https://design-system.service.gov.uk/components/ does, so perhaps nicest for us to follow?